src/main/java/net/snowflake/client/jdbc/telemetryOOB/TelemetryService.java

```diff
@@ -253,7 +257,9 @@ private enum TELEMETRY_API {
         "kyTKLWpEZSaJnrzTZ63I96QXZHKsgfqbaGmAaIWf"), // pragma: allowlist secret
     PROD(
         "https://client-telemetry.snowflakecomputing.com/enqueue",
-        "wLpEKqnLOW9tGNwTjab5N611YQApOb3t9xOnE1rX"); // pragma: allowlist secret
+        "wLpEKqnLOW9tGNwTjab5N611YQApOb3t9xOnE1rX"), // pragma: allowlist secret
```

Comment:
Are these real secrets? will it expire? will it cause any billing issue as code is open source?

Reply:
They've been around for a very long time and have been security reviewed. Also not related to the change. That said, if its concern lets open a separate ticket for that.